engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -245,7 +245,7 @@ public boolean isSynthetic() {
    * @return true iff this module is private (project-private).
    */
   public boolean isPrivate() {
-    return ir.isPrivate();
+    return ir == null || ir.isPrivate();
```

Comment:
Why should `isPrivate` be `true` for synthetic modules?

(Not saying it shouldn't but I would like understand the rationale)

Reply:
Well.. they are either private or not. And I don't see any use-cases for making them public, especially given that there is no source code/documentation available for them for the users and we don't give any guarantees how those modules are being constructed.

I believe @Akirathan's had a similar intuition that they should be private.

Reply:
Rule of a thumb is: The more stuff we can make private, the better. Unless something breaks. And _synthetic_ modules certainly feel like they should be private.